guides/Other Subjects/CreatingPointsSystems.md

```diff
@@ -0,0 +1,161 @@
+Creating a points system (also known as **Social Module**) in your bot is quite easy in Klasa, as we will use our configuration system: **SettingGateway**. You do NOT need to rewrite any of your code to change the provider, but we also suggest to **NOT** use the JSON provider for this in production, as you may reach the limit of files you can open simultaneously in your [OS](https://en.wikipedia.org/wiki/Operating_system).
+
+## Setting the schema
+
+Before we work with the social module, we need to update the built-in {@link GatewayDriver#users users' gateway} to implement a new key:
+
+```javascript
+async function init() {
+	if (!this.client.gateways.users.schema.has('experience')) {
+		this.client.gateways.users.schema.add('experience', {
+			type: 'integer',
+			default: 0,
+			configurable: false
+		});
+	}
+}
+```
+
+Where we'd check if the schema has the key `experience`. If it doesn't, we add it as a new key, with type `integer` (doubtfully we'll use `float` on this) and make it unconfigurable for the built-in userconf command so the end users do not cheat by modifying their stats.
+
+> **Note**: The name of the key can be any, can be `xp`, `points`... but we will use this one for this guide.
+
+## Setting the monitor
+
+Now that we have set up the schema, we will want to create a monitor
+
+```javascript
+const { Monitor } = require('klasa');
+
+module.exports = class extends Monitor {
+
+	constructor(...args) {
+		super(...args, {
+			enabled: true,
+			ignoreBots: true,
+			ignoreSelf: true,
+			ignoreOthers: false
+		});
+	}
+
+	async run(msg) {
+		// If the message was not sent in a TextChannel, ignore it.
+		if (!msg.guild) return;
+
+		// Update the user's configuration entry by adding 1 to it.
+		await msg.author.configs.update('experience', msg.author.configs.experience + 1);
+	}
+
+};
+
+```
+
+Alternatively, we can create the `init` method and ensure the users' schema always has our key.
+
+## Level up!
+
+Some social bots have level up messages, but, how do we set it up? There are two ways to achieve this:
+
+1. We calculate the current level and the next level on the fly. This system is, however, harder to implement and it processes a lot of maths, but it's also RAM friendly for massive bots. We won't cover this in this guide.
```

Comment:
Again, double this; "We won't cover this in this guide". Maybe change to "We won't cover this in the guide".